pkg/operator/ceph/config/livenessprobe.go

```diff
@@ -39,10 +39,21 @@ func ConfigureLivenessProbe(daemon rookv1.KeyType, container v1.Container, healt
 	if _, ok := healthCheck.LivenessProbe[daemon]; ok {
 		if !healthCheck.LivenessProbe[daemon].Disabled {
 			probe := probeFnMap[daemon](healthCheck)
-
 			// If the spec value is empty, let's use a default
 			if probe != nil {
 				container.LivenessProbe = probe
+				// In case of any empty field of probe struct, we are setting default values.
+				if probe.FailureThreshold == 0 {
+					container.LivenessProbe.FailureThreshold = 3
+				} else if probe.PeriodSeconds == 0 {
```

Comment:
I believe we want `if` instead of `else if` so all the conditions can be applied, not just the first one.
```suggestion
				}
				if probe.PeriodSeconds == 0 {
```